framework/.classpath

```diff
@@ -11,7 +11,5 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry combineaccessrules="false" exported="true" kind="src" path="/dataflow"/>
 	<classpathentry combineaccessrules="false" exported="true" kind="src" path="/javacutil"/>
-	<classpathentry combineaccessrules="false" kind="src" path="/annotation-file-utilities"/>
```

Comment:
Why are these being removed?
`checker-framework/framework/src/org/checkerframework/common/wholeprograminference/AnnotationConverter.java` and other classes in `wholeprograminference` use those projects.